airflow-core/src/airflow/dag_processing/bundles/manager.py

```diff
@@ -81,6 +83,54 @@ def _add_example_dag_bundle(config_list):
     )
 
 
+def _is_safe_bundle_url(url: str) -> bool:
```

Comment:
The documentation should mention these restrictions.

Reply:
Done